### PR TITLE
add the cffi recipes

### PR DESCRIPTION
**Pull request by vgauthier**

### CODE REVIEW

File: recipes/cffi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "cffi" %}
+{% set version = "1.8.3" %}
+{% set md5 = "c8e877fe0426a99d0cf5872cf2f95b27" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
```
Comment (scopatz) [inline recipes/cffi/meta.yaml]: Please remove comments

**Comment by jakirkham** [comment]
There's another PR ( https://github.com/conda-forge/staged-recipes/pull/1338 ) for `cffi`. Maybe we can consolidate the efforts. 😉

**Comment by jakirkham** [comment]
cc @goanpeca

**Comment by goanpeca** [comment]
fine by me ;')

**Comment by synapticarbors** [comment]
I think that this PR should probably be closed in favor of #1338 since the older PR is in much closer shape to being mergeable. This PR is missing a number of dependencies (namely `pycparser` and `libffi`) and would likely converge to something that looks very close to #1338. I'd be very supportive of @vgauthier being added as co-maintainer on the older PR.

**Comment by jakirkham** [comment]
Agreed. @vgauthier , sound good to you?

**Comment by vgauthier** [comment]
@jakirkham @goanpeca @synapticarbors Soons good to me

**Comment by synapticarbors** [comment]
Closing in favor of further development on #1338

**Pull request closed without merging**